graphics/nanovg/Portfile

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+set shorthash       2bead03
+set fullhash        2bead03bea43b2418060aaa154f972829995e663
+set lastcommit      20200320
+github.setup        memononen nanovg $shorthash
```

Comment:
do you know if upstream plan on making any proper releases any time ?

Reply:
It doesn't look like it, based on their GitHub project pages ("0 releases"). :(